spot-client/src/common/backend/SpotBackendService.js

```diff
@@ -217,13 +224,30 @@ export class SpotBackendService extends Emitter {
         }
 
         if (!registerDevicePromise) {
+            const storedEndpointId = persistence.get(this.endpointIdPersistenceKey);
```

Comment:
Are there any cases where the endpointIdPersistenceKey needs to be cleared? The point of it is not to be cleared?

Reply:
It seems that this one should not be cleared.

Reply:
At least at this point I'm not aware of any cases when it should be cleared.